custom-login/browserslist

```diff
@@ -0,0 +1,12 @@
+# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
+# For additional information regarding the format and rule options, please see:
+# https://github.com/browserslist/browserslist#queries
+
+# You can see what browsers were selected by your queries by running:
+#   npx browserslist
+
+> 0.5%
+last 2 versions
+Firefox ESR
+not dead
+not IE 9-11 # For IE 9-11 support, remove 'not'.
```

Comment:
Shouldn't we default to match our official supported browsers?

Reply:
this was generated by the ng update, but seems good. This would restrict to IE Edge+

Reply:
Per our offline discussion, it makes more sense to not change this until we have testing to ensure the change is accurate.